operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -341,17 +350,19 @@ private boolean shouldReconcile(Deployment d) {
             log.errorf("Wrong number of containers for Deployment %s/%s", d.getMetadata().getNamespace(), d.getMetadata().getName());
             return false;
         }
-        return !containers.get(0).getResources().equals(deploymentResourceRequirements);
+        Container ingressContainer = containers.get(0);
+        return !(ingressContainer.getResources().equals(deploymentResourceRequirements) && Objects.equals(ingressContainer.getCommand(), ingressContainerCommand.orElse(null)));
     }
 
     private void doIngressPatch(Deployment d) {
-        log.infof("Updating the resource limits for Deployment %s/%s", d.getMetadata().getNamespace(), d.getMetadata().getName());
+        log.infof("Updating the resource limits/container command for Deployment %s/%s", d.getMetadata().getNamespace(), d.getMetadata().getName());
         openShiftClient.apps().deployments().inNamespace(d.getMetadata().getNamespace())
             .withName(d.getMetadata().getName()).edit(
                 new TypedVisitor<ContainerBuilder>() {
                     @Override
                     public void visit(ContainerBuilder element) {
                         element.withResources(deploymentResourceRequirements);
+                        element.withCommand(ingressContainerCommand.orElse(null));
```

Comment:
Just to make sure, this isn't a part of the deployment that the controller is using to compute its modification hash from right - that is we can modify it and the ingress operator won't immediately try to flip it back.